src/components/AlertMessage/index.js

```diff
@@ -0,0 +1,53 @@
+import React, { useEffect } from 'react';
+
+import { useDispatch, useSelector } from 'react-redux';
+
+import { toggleSnackbarClose } from './alertRedux/actions';
+import { Container, CrossIcon, SuccessMessage, BTN, SuccessIconButton, ErrorIconButton } from './style';
+
+const Snackbar = ({ timeout, type }) => {
+  const dispatch = useDispatch();
+  const {
+    uiReducer: { toggleSnackbar: SHOW, snackbarMessage: MESSAGE },
+  } = useSelector((state) => state);
```

Comment:
This is not the right way to write reducer for useSelector please have a look at other useSelectors..